.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+
+<!-- 
+
+Sorry to hear you're having trouble! If you have a question, please ask in 
+[SO or gitter](https://github.com/callemall/material-ui/blob/master/CONTRIBUTING.md#asking-questions)
+
+If you are having an issue with click events, please re-read the 
+[README](https://github.com/callemall/material-ui#react-tap-event-plugin) (you did read the README, right?).
+
+Questions, or issues that don't provide sufficient information, may be closed without comment. 
+(We're sure you'd rather we work on improving Material-UI than chasing ghost issues.)
+
+If you think you have found a _new_ issue that hasn't already been reported, please complete the template below.
+
+-->
+
+Problem Description
+===================
+<!-- Please include steps to reproduce the problem, code sample, screenshots as appropriate. -->
+
+Versions
+========
+<!-- Versions affected. (Please test with Material-UI HEAD.) -->
+- Material-UI:
+- React:
+- Browser:
+- OS:
```

Comment:
Makes sense. I was thinking of cases where the same browser differs across OSes (scrollbar behaviour for example), but it's uncommon enough that we can ask if we need it.